datasette/my_example.py

```diff
@@ -12,8 +12,6 @@
 
 
 def initialize_db(timestamp: datetime) -> datetime:
-    print(f"Inside the docker file at - {timestamp}")
-    log.debug("This is a debugging message")
     db = sqlite_utils.Database("laliga1819.db")
     db["laliga1819"].insert_all(
```

Comment:
we can test that the requests.get call is made using a mock of request and using patch in the test case.
patch is used to replace the actual request object with a fake version that we can control (and has no link to the external url).
resources about mock and patch https://realpython.com/python-mock-library/#patch